### PR TITLE
monetdb 11.39.5

### PR DESCRIPTION
**Pull request by monetdb-team**
Autogenerated PR as part of the release

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> brew audit monetdb --online --git --skip-style
==> FAILED
Error: 1 problem in 1 formula detected
monetdb:
  * Files were found with references to the Homebrew shims directory.
    The offending files are:
      bin/mserver5
```

File: Formula/monetdb.rb [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
+                      "-DGEOM=OFF",
+                      "-DNETCDF=OFF",
+                      "-DODBC=OFF",
+                      "-DPY3INTEGRATION=OFF",
+                      "-DRINTEGRATION=OFF",
+                      "-DSHP=OFF",
+                      "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}",
+                      "-DREADLINE_ROOT=#{Formula["readline"].opt_prefix}"
+      # remove reference to shims directory from compilation/linking info
+      system "sed", "-i.bak", "/printf/s|\"/[^ ]*/|\"|",
```
Comment (SMillerDev) [inline Formula/monetdb.rb:40]: I think `inreplace` should be used here.

**Comment by SMillerDev** [comment]
Linkage failure on
```
  /usr/local/opt/xz/lib/liblzma.5.dylib (xz)
```

**Comment by dtrodrigues** [comment]
Can this build with python 3.9?

**Comment by sjoerdmullender** [comment]
Yes, Python is only used to do some small processing during the build process.

**Comment by sjoerdmullender** [comment]
And the only reason Python 3 is mentioned as a requirement is because old systems don't appear to have one (or not a new enough one).

**Comment by dtrodrigues** [comment]
Once the formula is updated to depend on 3.9 to build this should be good to go.

**Review by dtrodrigues — APPROVED** [review]
Thanks for the update and contribution to homebrew @sjoerdmullender

**Pull request closed without merging**